src/env.h

```diff
@@ -64,6 +56,18 @@ namespace node {
 #define NODE_PUSH_VAL_TO_ARRAY_MAX 8
 #endif
 
+// PER_ISOLATE_* macros: we're going slightly crazy with macros here but the
+// end hopefully justifies the means. We have a lot of per-isolate properties
+// and adding and maintaining their getters and setters by hand would be
+// a nightmare so let's make the preprocessor generate them for us.
+//
+// In each macro, `V` is expected to be the name of a C function which accepts
+// the number of arguments provided in each tuple in the macro body, typically
+// two. The named function will be invoked against each tuple.
```

Comment:
I guess it could technically be anything which could syntactically be followed by a [parameter list](http://en.cppreference.com/w/cpp/language/function#Parameter_list)? I'll rephrase.